.gitignore

```diff
@@ -2,3 +2,4 @@ composer.phar
 /vendor/
 *.ignored
 **/IgnoredTest.php
+.idea
```

Comment:
@dodozhang21 

As @covenantd pointed out, `.gitignore` should not be polluted with rules for IDEs - how about ignoring it globally?

For reference, see https://help.github.com/articles/ignoring-files/#create-a-global-gitignore.